### ISSUE TITLE
Add a message when attacking energy ghosts with non-enchanted sword.

### ISSUE THREAD

**Issue by rwtema** [rwtema/Dark-Menagerie#4]
Energy ghosts are only vulnerable to enchanted weapons (or splash potions). Warn the player if he tries to attack with a non-enchanted weapon.

**Comment by therealericbj** [comment]
Yes, please.   ...and the 5 other players on my server also say "Yes, please".
